### ISSUE TITLE
PHP Fatal error:  Unsupported operand types in vendor/robbiep/cloudconvert-laravel/src/RobbieP/CloudConvertLaravel/HttpClientAdapter.php on line 31

### ISSUE THREAD

**Issue by andresnijder** [robbiepaul/cloudconvert-laravel#2]
occurs: 
CloudConvert::file($path)
->callback(url() . '/api/notifications/convert/videos')
->to('ogg');

**Comment by robbiepaul** [comment]
Can you provide the contents of $path?

**Comment by andresnijder** [comment]
$path = 'http://cdn.vellance.com/superhero/site/content/uploads/project/small_54735848bc3f8.mp4';

**Comment by robbiepaul** [comment]
What version of php are you using?

Also the syntax for using callbacks should be:

```
CloudConvert::file($path)
->callback(url() . '/api/notifications/convert/videos')
->convert('ogg');
```

Because `to()` attempts to save the file when it isn't ready. The save function should be called in the callback script.

Hope that helps

**Comment by andresnijder** [comment]
I'm using PHP 5.5.14-1~dotdeb.1
I get the same error when i use 'convert'.

**Comment by robbiepaul** [comment]
I've applied a fix in the latest update: 3a275f854f3297d2b32547de81648c172e391302
Try a `composer update robbiep/cloudconvert-laravel`

**Comment by andresnijder** [comment]
thanks! This works!
